test/internal/layers/layerfolders.go

```diff
@@ -67,6 +67,7 @@ func LayerFolders(tb testing.TB, imageName string) []string {
 	return imageLayers[imageName]
 }
 
+// Deprecated: This relies on docker. Use [FromChainID] or [FromMount] instead.
```

Comment:
Do we still need to keep these functions? Or can we remove these and just your new LazyLayers that you added everywhere?

Reply:
done